### PR TITLE
docs(CONTRIBUTING): fixes cypress version

### PR DESCRIPTION
**Pull request by Fabs**
Add the newest version of cypress to the documentation

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -351,14 +351,13 @@ To setup cypress you need to follow the following steps:
 
 We rely on a very specific version of cypress, and cypress desktop app.
 ```sh
-  npm install -g cypress-cli@0.14.0
-  cypress install —cypress-version 0.19.1
+  npm install cypress@1.4.1
```
Comment (nLight) [inline CONTRIBUTING.md]: It's a dependency now, no need installing it anymore.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -351,14 +351,13 @@ To setup cypress you need to follow the following steps:
...
 ```
 
 3. Open Cypress.
 
   ```sh
-  cypress open
+  ./node_modules/.bin/cypress open
```
Comment (nLight) [inline CONTRIBUTING.md]: We also can just use `npm run cypress` that will do `cypress run` not the same but just runs tests :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (Fabs) [reply CONTRIBUTING.md]: Sure, I also showed how to install dev dependencies if you haven't yet.

File: CONTRIBUTING.md (hunk 2 above)
Comment (Fabs) [reply CONTRIBUTING.md]: I think we should have instructions for both. I am fixing accordingly.

**Comment by Fabs** [comment]
fixed. @nLight and @DanielMSchmidt , could u look again?

**Comment by nLight** [comment]
run system tests

**Pull request merged**